pom.xml

```diff
@@ -20,7 +20,7 @@
 
     <dep.jackson.version>2.11.0</dep.jackson.version>
     <dep.jackson-databind.version>2.11.0</dep.jackson-databind.version>
-    <dep.protobuf-java.version>3.8.0</dep.protobuf-java.version>
+    <dep.protobuf-java.version>3.19.3</dep.protobuf-java.version>
```

Comment:
What's the context around this dep change?

Reply:
Version 3.8.0 doesn't work on M1 MacBooks, somewhere around ~3.17 they started publishing mac-arm64 protoc binaries